component/generated-rolebindings.jsonnet

```diff
@@ -38,6 +49,37 @@ local generateDefaultRolebindingInNsPolicy = kyverno.ClusterPolicy('default-role
           },
         },
       },
+      {
+        name: 'patch-uninitialized-default-rolebinding',
+        match: common.MatchRoleBindings(
+          names=[ params.generatedDefaultRoleBindingInNewNamespaces.bindingName ],
+          selector={
+            matchLabels: {
+              'appuio.io/uninitialized': 'true',
+            },
+          },
+        ),
+        context: [
+          {
+            name: 'organization',
+            apiCall: {
+              urlPath: '/api/v1/namespaces/{{request.object.metadata.namespace}}',
+              jmesPath: 'metadata.labels."appuio.io/organization"',
+            },
+          },
+        ],
+        mutate: {
+          patchesJson6902: |||
```

Comment:
I'd prefer native jsonnet here. there's `std.manifestJson` and `std.manifestYamlDoc` available

Reply:
Good point! I didn't like it either, but didn't think of using `std.manifestJson`

Reply:
I'd strongly prefer to have this with `std.manifestJson()`, can you please update the implementation accordingly?